src/lexer.coffee

```diff
@@ -416,13 +416,13 @@ exports.Lexer = class Lexer
         @token 'REGEX_START', '(',    {length: 0, origin}
         @token 'IDENTIFIER', 'RegExp', length: 0
         @token 'CALL_START', '(',      length: 0
-        @mergeInterpolationTokens tokens, {double: yes, heregex: {flags}, endOffset: end}, (str) =>
+        @mergeInterpolationTokens tokens, {double: yes, heregex: {flags}, endOffset: end - flags.length, quote: '///'}, (str) =>
```

Comment:
Allow the `TemplateLiteral` (ie `StringWithInterpolations`) generated in the AST for a heregex to have `quote: '///'`

The `endOffset` change excludes flags location data from the location data for the `TemplateLiteral`, which makes sense (the `TemplateLiteral` corresponds to `///...///`, not the trailing flags)